README.md

```diff
@@ -175,6 +175,12 @@ end
 
 ---
 
+## Sorbet
+
+We make use of ![Sorbet](https://sorbet.org/) in cli-ui. We provide stubs for Sorbet so that you can use this gem even
```

Comment:
I think `!` is only needed for image references, but I could be mistaken. Otherwise looks good